app/assets/javascript/controllers/reassignment_buttons_controller.js

```diff
@@ -1,6 +1,5 @@
 import { Controller } from "@hotwired/stimulus";
 import $ from "jquery";
-import "selectize/dist/js/selectize";
```

Comment:
no idea why we did not remove it earlier, unused import

Reply:
It looks like the Hub has the same problem